config/initializers/new_framework_defaults_6_0.rb

```diff
@@ -0,0 +1,45 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Are we planning on using this file at all? If not is it useful to keep around once the upgrade is complete?

Reply:
I've removed it.